v2/client.go

```diff
@@ -202,6 +214,29 @@ func (c *client) handleFailureResponse(response *http.Response) error {
 	}
 }
 
+func buildOriginatingIdentityHeaderValue(i *AlphaOriginatingIdentity) (string, error) {
+	if i == nil {
+		return "", nil
+	}
+	if i.Platform == "" {
+		return "", errors.New("originating identity platform must not be empty")
+	}
+	if i.Value == "" {
+		return "", errors.New("originating identity value must not be empty")
+	}
+	if err := isValidJSON(i.Value); err != nil {
+		return "", errors.New(fmt.Sprintf("originating identity value must be valid JSON: %v", err))
```

Comment:
nit: you can use `fmt.Errorf("originating...", err)` instead and save precious bytes :-)